### DISCUSSION TITLE
show pygubu-designer in the app-repository from the computer

### DISCUSSION THREAD

**Discussion by ghost** [alejandroautalan/pygubu-designer#247] [Ideas]
up untill now you can start pygubu-designer only through typing its name in the terminal. It should be nice when the programm can be selected from the list of applications on my computer. It is not only nicer, it is also what the user expects.

**Comment by carsten-engelke** [comment]
I installed pygubu-designer with pipx and simply put a shortcut in my StartMenu folder linking it to the pygubu-designer.exe in "pipx\venvs\pygubu-designer\Scripts\pygubu-designer.exe". Does this answer your question?
